plugins/star-rating/api/api.js

```diff
@@ -110,14 +452,87 @@ var plugin = {},
                         });
                         common.returnOutput(params, result);
                         return true;
-                        //common.returnMessage(params, 200, result);
                     }
                 }
             );
             return true;
         }
         return false;
     });
+    
+    plugins.register("/i/apps/create", function(ob){
+        var params = ob.params;
+        var appId = ob.appId;
+        common.db.collection('web_feedback_widgets_' + appId).insert({_id:"meta"},function(){});
+        common.db.collection('feedback' + appId).ensureIndex({"ts":1, "uid":1}, {unique:true}, function(){});
```

Comment:
Here I think we need separate indexes and not compound, cause we are either fetch by ts or by uid (whem merging, deleting, etc) not by both.
Also definitely not needed unique constraint